week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -41,19 +41,30 @@ console.log(twoDimensionalArray[1][2]);
 
 // Examples accessing to array element through array methods
 // Example 1
-let moreThanTwoArrays = twoDimensionalArray.filter(embeddedArray => embeddedArray.length > 2);
+let moreThanTwoArrays = twoDimensionalArray.filter(
+  (embeddedArray) => embeddedArray.length > 2
+);
 console.log(moreThanTwoArrays);
 // This filter only keeps embededd arrays that have more than 3 elements
 
 // Example 2
-let arrayLengths = twoDimensionalArray.map(embeddedArray => embeddedArray.length);
+let arrayLengths = twoDimensionalArray.map(
+  (embeddedArray) => embeddedArray.length
+);
 console.log(arrayLengths);
 // This gives back how many elements of each array have
 
 /* + + + + + + + */
 /* Now try the exercise */
 
-function getEligibleStudents() {
+function getEligibleStudents(arr) {
+  let arr1 = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i][1] >= 8) {
+      arr1.push(arr[i][0]);
+    }
+  }
+  return arr1;
```

Comment:
while this works fine, we would like you to think about a way to solve it using array methods (you can use them inside each other)